browser-raw.js

```diff
@@ -76,9 +76,10 @@ function flush() {
 
 // Safari 6 and 6.1 for desktop, iPad, and iPhone are the only browsers that
 // have WebKitMutationObserver but not un-prefixed MutationObserver.
-// Must use `global` instead of `window` to work in both frames and web
-// workers. `global` is a provision of Browserify, Mr, Mrs, or Mop.
```

Comment:
Please retain an explanation of the use of global or self and the environments they support, specifically WebPack (browserify and my https://github.com/gutentags/system both inject a global alias to simplify portability).